### ISSUE TITLE
Canvas native bindings

### ISSUE THREAD

**Issue by dceejay** [yhwang/node-red-contrib-post-object-detection#5]
Hi,

One of the pre-reqs you have is canvas which has native c++ bindings... These do not recompile on platforms like the Raspberry PI so we can't use this node there... 

Have you seen the pureimage package https://www.npmjs.com/package/pureimage ? That is pure JS so could maybe be more cross platform.

Just a thought. I know it would mean a re-write so may not be feasible/wanted. Thanks

**Comment by yhwang** [comment]
good info. I did manually compile canvas on arm earlier. I will try it out.

**Comment by dceejay** [comment]
Yes it can be done manually - but is always a barrier for users.
